docs/modules/policies/pages/authoring_tips.adoc

```diff
@@ -2,17 +2,45 @@ include::ROOT:partial$attributes.adoc[]
 
 :policy-schema-file: /policy/v1/Policy.schema.json
 :test-schema-file: /policy/v1/TestSuite.schema.json
+:testdata-principals-schema-file: /policy/v1/TestFixture/Principals.schema.json
+:testdata-resources-schema-file: /policy/v1/TestFixture/Resources.schema.json
+:testdata-auxdata-schema-file: /policy/v1/TestFixture/AuxData.schema.json
+
 :current-policy-schema-url: {app-schema-url-current}{policy-schema-file}
 :latest-policy-schema-url: {app-schema-url-latest}{policy-schema-file}
 :current-test-schema-url: {app-schema-url-current}{test-schema-file}
 :latest-test-schema-url: {app-schema-url-latest}{test-schema-file}
+:current-testdata-principals-schema-url: {app-schema-url-current}{testdata-principals-schema-file}
+:latest-testdata-principals-schema-url: {app-schema-url-latest}{testdata-principals-schema-file}
+:current-testdata-resources-schema-url: {app-schema-url-current}{testdata-resources-schema-file}
+:latest-testdata-resources-schema-url: {app-schema-url-latest}{testdata-resources-schema-file}
+:current-testdata-auxdata-schema-url: {app-schema-url-current}{testdata-auxdata-schema-file}
+:latest-testdata-auxdata-schema-url: {app-schema-url-latest}{testdata-auxdata-schema-file}
+
+
 
 = Policy authoring
 
 == Tips for working with policies
 
 * Policies can be in either YAML or JSON formats. Accepted file extensions are `.yml`, `.yaml` or `.json`. All other extensions are ignored.
-* The JSON schema for Cerbos policies is available at `{current-policy-schema-url}` and the xref:compile.adoc[test suite] schema is available at `{current-test-schema-url}`. If you prefer to always use the latest version, they can be accessed at `{latest-policy-schema-url}` and `{latest-test-schema-url}` respectively.
+* The JSON schema for Cerbos files are avaliable at:
+** Policy
+*** `{current-policy-schema-url}` 
+** xref:compile.adoc[Test suite]
+*** `{current-test-schema-url}`
+** xref:compile.adoc#_sharing_test_fixtures[Principal test fixtures]
```

Comment:
I think rather than using the autogenerated anchor, we should add a custom one `#fixtures` so that if the wording in the heading changes it doesn't break the link (we do this for `#testing` already)